crates/cadenza-eval/src/ir/types.rs

```diff
@@ -292,18 +295,16 @@ impl std::fmt::Display for IrInstr {
                 args,
                 ..
             } => {
+                // Function calls with juxtaposition
                 if let Some(res) = result {
-                    write!(f, "{}: {} = call {}(", res, ty, func)?;
+                    write!(f, "let {}: {} = {}", res, ty, func)?;
```

Comment:
I think we want to keep the `call` so we minimize the number of callable things in the scope